apps/bors_github/lib/github/server.ex

```diff
@@ -390,6 +390,31 @@ defmodule BorsNG.GitHub.Server do
   end
 
   @doc """
+  Uses a token from the cache, or, if the request fails,
+  retry without using the cached token.
+  """
+  @spec use_token!(ttoken, ttokenreg, ((ttoken) -> term)) ::
+    {:reply, term, ttokenreg}
```

Comment:
I didn't know about `@spec`. Did you know it is used by https://github.com/jeremyjh/dialyxir ?